src/main/java/com/robin/hoteless/App/AddNewUser.java

```diff
@@ -0,0 +1,37 @@
+package com.robin.hoteless.App;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+public class AddNewUser {
```

Comment:
Your objects should be named with a noun, not a verb; the API itself is a verb, but that is a method. Just a standard naming note. :)